platform/core.windows/test/unit/src/org/netbeans/core/windows/view/ui/AutoHidingMenuBarManualTest.java

```diff
@@ -61,6 +62,11 @@ public AutoHidingMenuBarManualTest() {
         initComponents();
     }
 
+    @Test
+    public void runMain() {
+        main();
```

Comment:
No, this file should _not_ be run automatically during a build--it's intended for manual use only. (See comment.)